lib/ts/recipe/emailverification/emailVerifiedClaim.ts

```diff
@@ -0,0 +1,48 @@
+import EmailVerificationRecipe from "./recipe";
+import { BooleanClaim } from "../session/claims";
+import { SessionClaimValidator } from "../session";
+
+export class EmailVerifiedClaimClass extends BooleanClaim {
+    constructor() {
+        super({
+            key: "st-ev",
+            async fetchValue(userId, userContext) {
+                const recipe = EmailVerificationRecipe.getInstanceOrThrowError();
+                let email = await recipe.getEmailForUserId(userId, userContext);
+                return recipe.recipeInterfaceImpl.isEmailVerified({ userId, email, userContext });
+            },
+        });
+
+        this.validators = {
+            ...this.validators,
+            isValidated: (minRefetchDelayInSeconds: number = 10) => ({
```

Comment:
A better name would be `isVerified`? Cause it's email verification, and not email validation